### PR TITLE
Lair config fixes TK-05044

### PR DESCRIPTION
**Pull request by zippy**
Updated config to have keystore path, and renames passphrase in config from "Mock" to "FromConfig"

### CODE REVIEW

File: crates/holochain/src/conductor/config.rs [hunk 1]
```diff
@@ -195,11 +200,45 @@ pub mod tests {
...
                 use_dangerous_test_keystore: true,
             }
         );
     }
+
+    #[test]
+    fn test_config_keystore() {
+        let toml = r#"
+    environment_path = "/path/to/env"
+    use_dangerous_test_keystore = true
```
Comment (freesig) [inline crates/holochain/src/conductor/config.rs:216]: Just wondering about this? Is the point of this PR to still be using the test keystore?

Comment (freesig) [reply crates/holochain/src/conductor/config.rs:216]: Might be off topic for this  PR though

File: crates/holochain/src/conductor/config.rs [hunk 2]
```diff
@@ -195,11 +200,45 @@ pub mod tests {
...
+
+    keystore_path = "/path/to/keystore"
+
+    [passphrase_service]
+    type = "fromconfig"
+    passphrase = "foobar"
```
Comment (freesig) [inline crates/holochain/src/conductor/config.rs:222]: Also might be that I'm missing context but this is just for testing right? We're not actually storing passwords in text files?

**Review by freesig — APPROVED** [review]
Assuming this is just for testing it looks good to me

File: crates/holochain/src/conductor/config.rs (hunk 2 above)
Comment (neonphog) [reply crates/holochain/src/conductor/config.rs:222]: We're totally storing passwords in text files : )

This is for the holoports - they need to be able to log into lair automatically when they reboot... so for now we're putting that unlock passphrase in the config file.

Someday it'd be nice to have a whole ecosystem where we could ping someone's connected cell phone when the holoport reboots and they could remotely enter the passphrase... but we're not there yet.

File: crates/holochain/src/conductor/config.rs (hunk 1 above)
Comment (neonphog) [reply crates/holochain/src/conductor/config.rs:216]: For the unit tests we want to use the test keystore - holoport configs will not set this to true

**Review by neonphog — APPROVED** [review]
:+1:

**Review by peeech — CHANGES_REQUESTED** [review]
The `keystore_path` part looks great. I have doubts about the way passphrase is stored though. See comments.

**Comment by zippy** [comment]
Yes, I think we could add a add a "FromFile" passphrase service too, but I'd like that to be a separate PR later.

**Pull request merged**